roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
@rhcarvalho Thanks! Interesting...
Let me ask you then how exactly this works (/me confused a bit 😮 ).

- If `timeoutSeconds` is not present, is any default values used for it instead?
- Does it mean that readiness probe works the same regardless `timeoutSeconds` being setup at all?
- If the `command` points to shell script which has [internal timeout](https://github.com/openshift/origin-aggregated-logging/pull/358/files#diff-e0db40ac0a1f0750c954836bcba1c905) (specifically: `curl --max-time ...`) does it make sense to bother with probe `timeoutSeconds` at all? Can we just rely on the internal script timeout instead?